e2e-tests/kuttl/scheduled-backup/00-priority-class.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: scheduling.k8s.io/v1
```

Comment:
Does it allow two test steps with same index? We have `00-basic-startup.yaml ` and `00-priority-class.yaml`.

Reply:
Yeap. We are allowed to explicitly state TestStep only once per index, and as many as we want kubernetes manifests and files under that index